int64 startStop
---
int64 status
